src/content/docs/apm/transactions/transaction-traces/transaction-traces-trace-details-page.mdx

```diff
@@ -144,43 +130,34 @@ In the **Drilldown** column, icons indicate that additional data is available:
 
       <td>
         The database <Icon name="fe-database"/>
-        icon links to the raw database query for that query segment. Depending on your [transaction trace settings](/docs/features/configuring-transaction-traces), this query can either show as raw data or obfuscated data. If the total time for the segment exceeds the query plan threshold in your settings, the query analysis also will appear, along with a stack trace showing the exact location of the call in your code.
+        icon identifies a database call segment. Click on the segment to view the raw database query. Depending on your [transaction trace settings](/docs/features/configuring-transaction-traces), this query can either show as raw data or obfuscated data.
       </td>
     </tr>
 
     <tr>
       <td>
-        <img title="icon-search" alt="icon-search" src={apmIconParameters}/> Stack traces
+        <img title="icon-stack-trace" alt="icon-stack-trace" src={apmIconStackTrace} width="35px" height="35px" /> Stack traces
       </td>
 
       <td>
-        The magnifying glass <Icon name="fe-search"/>
-        links to stack trace details.
+        The magnifying glass <Icon name="nr-stack-trace"/>
```

Comment:
I think this text needs updated since we replaced the magnifying glass icon. Otherwise, these look good to me.

Reply:
good catch - fixed!